discovery/kubernetes/kubernetes.go

```diff
@@ -40,20 +40,24 @@ import (
 const (
 	// kubernetesMetaLabelPrefix is the meta prefix used for all meta labels.
 	// in this discovery.
-	metaLabelPrefix = model.MetaLabelPrefix + "kubernetes_"
-	namespaceLabel  = metaLabelPrefix + "namespace"
+	metaLabelPrefix  = model.MetaLabelPrefix + "kubernetes_"
+	namespaceLabel   = metaLabelPrefix + "namespace"
+	metricsNamespace = "prometheus_sd_kubernetes"
 )
 
 var (
+	// DefaultSDConfig is the default Kubernetes SD configuration
```

Comment:
(nit) I wouldn't change the position of this code unless there's a reason related to the PR.